src/js/web-components/video-download/VideoDownloader.js

```diff
@@ -180,6 +180,19 @@ export default class VideoDownloader extends HTMLElement {
     return subtitlesUrls;
   }
 
+  /**
+   * Returns the artwork images URLs used by Media Session API
+   * to render the media popup / notification.
+   *
+   * @returns {string[]} URLs.
+   */
+  getMediaSessionArtworkUrls() {
+    const artworkObjects = this.internal.videoData['media-session-artwork'] || [];
```

Comment:
Shouldn't we have default artwork URL there as a fallback or is it safe to assume `MEDIA_SESSION_DEFAULT_ARTWORK` will always be in the cache?

Reply:
@beaufortfrancois You're right, we should have the default artwork pulled in here. Updated the implementation in ab8c10422441e7c616fcb9b9158199a410cef00c.